apps/backend/src/mcp/tools/container_management.py

```
@@ -776,98 +752,106 @@ async def get_service_dependencies(
+                            for mount in mounts
+                            if mount.get("Type") == "bind" or mount.get("Type") == "volume"
+                        ],
                         "volume_mounts": [mount.get("Destination", "") for mount in mounts],
                         "labels": labels,
-                        "running": container_data.get("State", {}).get("Running", False)
+                        "running": container_data.get("State", {}).get("Running", False),
                     }
-                    
+
                     # Map networks to services
                     for network_name in network_settings.get("Networks", {}).keys():
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)